Casamari/MNC010.xml

```diff
@@ -122,77 +215,163 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                        <item xml:id="a1" corresp="#h2">
                            <locus from="2r" to="2v"/>
-                           <desc>The note begins as follows: 
-                              <q xml:lang="gez">
-                                 <lb n="1"/>ብፁዓን፡ እለ፡ የኀሡ፡ ስምዖ። 
-                                 <lb n="2"/>ወበኵሉ፡ ልቦሙ፡ የኃሥዎ። ኦሐሩ። 
-                                 <lb n="3"/>እለሰ፡ ይገብሩ፡ ዓመፃ፡ ፍኖተ፡ ዚአሁ፡ 
-                                 <lb n="4"/>እንተ፡ አዘዝከ፡ የዕቀቡ፡ ትእዛዘከ፡
-                              </q>
+                           <desc type="GuestText">
+                              <title ref="LIT2701Dawit">Excerpt from the Psalter</title>
                            </desc>
-                        </item> 
-                        </list>
+                           <note>The excerpt comprises <ref type="work"
+                                 corresp="LIT2701Dawit.118.2-14">Ps 118,2–4</ref>.</note>
+                           <q xml:lang="gez">
+                              <locus target="#2r"/>
+                              <seg part="I">
+                                 <lb n="1"/>ብፁዓን፡ እለ፡ የኀሡ፡ ስምዖ። <lb n="2"/>ወበኵሉ፡ ልቦሙ፡ የኃሥዎ። <lb
+                                    n="3"/>እለሰ፡ ይገብሩ፡ ዓመፃ፡ ፍኖተ፡ ዚአሁ፡ ኦሖሩ።<lb n="4"/>እንተ፡ አዘዝከ፡ ይዕቀቡ፡
+                                 ትእዛዘከ፡ ፈድፋደ።</seg>
+                           </q>
+                        </item>
+
+                        <item xml:id="e1">
+                           <desc> Scribal corrections occur interlineally (e.g. <locus
+                                 target="#4r #5v #7v #8r"/>), occasionally over erasure (e.g. <locus
+                                 target="#17r"/>. Erroneous characters or words are encircled with a
+                              red line, or with dots in red and black (e.g. <locus
+                                 target="#8r #23v #18r #34v #35v"/>).</desc>
+                        </item>
+
+                        <item xml:id="e2">
+                           <desc> The name of <foreign xml:lang="gez">ወይዘሮ፡ በትኃ፡</foreign> is
+                              written in pencil in the lower margin of <locus target="#2r"/>, the
+                              name of <foreign xml:lang="gez">ወይዘሮ፡ ፀሐይቱ፡</foreign> on <locus
+                                 target="#3v"/>.</desc>
+                        </item>
+
+                        <item xml:id="e3">
+                           <desc> Writing exercises in different hands were executed on <locus
+                                 target="#1r #2v #3r #3v #40v"/>, of the orders of the labiovelar
+                              characters <foreign xml:lang="gez">ኰ</foreign>, <foreign
+                                 xml:lang="gez">ጐ</foreign>, <foreign xml:lang="gez">ቈ</foreign>,
+                              and <foreign xml:lang="gez">ኰ</foreign> on <locus target="#1r"/>.
+                              Rough sketch of a <foreign xml:lang="gez"> ḥarag</foreign> in pen on
```

Comment:
Since we usually indicate ḥarags as an "ornamental band" in the decoDesc, I would avoid to introduce a new term here for coherence. What do you think? Otherwise, we can use `<foreign xml:lang="gez">ḥarag</foreign>` or `<foreign xml:lang="gez">ḥarag</foreign>-band` also in the decoDesc.